### ISSUE TITLE
Issue on rasberry pi 5 (and a workaround)

### ISSUE THREAD

**Issue by rossdargan** [badaix/snapcast#1257]
This is way above my knowledge, but I'm going to post the issue and how I fixed it in the hope it helps someone else.

I installed `snapclient_0.28.0-1_armhf-bookworm.deb`  on a rasberry pi 5 running ` 6.6.31+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.31-1+rpt1 (2024-05-29) aarch64 GNU/Linux`

Executing snapcast gave this error:
`snapclient: error while loading shared libraries: libavahi-common.so.3: ELF load command address/offset not page-aligned`

I fixed it by following this guide: https://forums.raspberrypi.com/viewtopic.php?t=358453

basically adding ` kernel=kernel8.img` to the config.txt file.

Sounds like its a 32bit/64bit issue.

Hope it helps someone in the future!

**Comment by badaix** [comment]
And why did you not install the 64 bit version [snapclient_0.28.0-1_arm64-bookworm.deb](https://github.com/badaix/snapcast/releases/download/v0.28.0/snapclient_0.28.0-1_arm64-bookworm.deb)?

**Comment by rossdargan** [comment]
yeah... good question. That does (rather obviously) fix it. The honest reason I didn't is because I wasn't sure that version would work on a pi...
